roles/kubernetes/control-plane/tasks/kubeadm-setup.yml

```diff
@@ -238,7 +238,8 @@
 
 # FIXME(mattymo): from docs: If you don't want to taint your control-plane node, set this field to an empty slice, i.e. `taints: {}` in the YAML file.
 - name: kubeadm | Remove taint for master with node role
-  command: "{{ bin_dir }}/kubectl --kubeconfig {{ kube_config_dir }}/admin.conf taint node {{ inventory_hostname }} node-role.kubernetes.io/master:NoSchedule- node-role.kubernetes.io/control-plane:NoSchedule-"
+  command: "{{ bin_dir }}/kubectl --kubeconfig {{ kube_config_dir }}/admin.conf taint node {{ inventory_hostname }} {{ item }}"
   delegate_to: "{{ groups['kube-master'] | first }}"
+  with_items: ["node-role.kubernetes.io/master:NoSchedule-", "node-role.kubernetes.io/control-plane:NoSchedule-"]
```

Comment:
I find
```
with_items:
  - "node-role.kubernetes.io/master:NoSchedule-"
  - "node-role.kubernetes.io/control-plane:NoSchedule-"
```
more readable and more common in Ansible roles
Can you squash you commits ?

Reply:
Sure, I just update the format as well as the author info.